### PR TITLE
[ADD] pos: how to add/modify customer notes from POS UI

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
Task ID: 2622090

### CODE REVIEW

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
+customer’s particular requests. For example, they do not want the product to be assembled, etc.
+
+Add customer notes
+==================
+
+You can add notes directly from an open :ref:`POS session <pos/start-session>`. To do so, click
+:guilabel:`Customer Notes` on the pad. Doing so opens a pop-up window on which you can add the
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/overview/customer_notes.rst:13]:
```suggestion
:guilabel:`Customer Note` on the pad. Doing so opens a pop-up window on which you can add the
```
It is spelled without an -S on my runbot

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   cart.
+
+.. image:: customer_notes/customer-notes-ui.png
+   :align: center
+   :alt: customer note button and notes (SO and POS session) on products in the cart
+
+Modify a customer note
+======================
+
+Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/overview/customer_notes.rst:27]:
```suggestion
Select the product from the cart to modify an existing note and click :guilabel:`Customer Note`.
```

Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/overview/customer_notes.rst:27]:
```suggestion
Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
```
To modify an existing note, select the product and click :guilabel:`Customer Note`.

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Doing so reopens the previous pop-up window. Alter the text and click :guilabel:`Add`.
+
+Receipts and invoices
+=====================
+
+The customer notes appear on customers' receipts and invoices similarly to how they appeared on the
+cart, meaning under the related product.
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/overview/customer_notes.rst:34]:
```suggestion
cart, under their related product.
```

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+.. image:: customer_notes/notes-receipt.png
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/overview/customer_notes.rst:36]: Not sure what purposes the notes serve if all they display is "Customer Note from SO". Maybe you should change it with something like "(This is where the customer note appears)"

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
+==============
+Customer notes
+==============
+
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/overview/customer_notes.rst:6]:
```suggestion
For instance, to provide cleaning and maintenance tips; they can also be used to track a
```

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+customer’s particular requests. For example, they do not want the product to be assembled, etc.
```
Comment (toaa-odoo) [inline content/applications/sales/point_of_sale/overview/customer_notes.rst:7]:
```suggestion
customer’s particular request, such as not wanting the product to be assembled for them.
```

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
A few points to consider

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+.. image:: customer_notes/customer-notes-ui.png
+   :align: center
+   :alt: customer note button and notes (SO and POS session) on products in the cart
+
+Modify a customer note
+======================
+
+Select the product from the cart to modify an existing note and click :guilabel:`Customer Notes`.
+Doing so reopens the previous pop-up window. Alter the text and click :guilabel:`Add`.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/overview/customer_notes.rst:28]: reopens and alter are correct

File: content/applications/sales/point_of_sale/overview/customer_notes.rst (hunk 6 above)
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/overview/customer_notes.rst:7]: Good catch, like the suggestion

File: content/applications/sales/point_of_sale/overview/customer_notes.rst [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+**Customer notes** allow you to provide customers with further information on a specific product.
+For instance, you can give them cleaning and maintenance tips, etc. They can also be used to track a
+customer’s particular requests. For example, they do not want the product to be assembled, etc.
+
+Add customer notes
+==================
+
+You can add notes directly from an open :ref:`POS session <pos/start-session>`. To do so, click
+:guilabel:`Customer Notes` on the pad. Doing so opens a pop-up window on which you can add the
+desired content for the note.
```
[the remark this answers is not in the document]
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/overview/customer_notes.rst:14]: I changed it a bit, but the suggestion would imply that there's content already and that you add more to it, which is not the case here

File: content/applications/sales/point_of_sale/overview/customer_notes.rst (hunk 3 above)
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/overview/customer_notes.rst:34]: I dismissed that suggestion, I think it looks fine the way it is

File: content/applications/sales/point_of_sale/overview/customer_notes.rst (hunk 5 above)
Comment (LoredanaLrpz) [reply content/applications/sales/point_of_sale/overview/customer_notes.rst:6]: I've added the first part, but left it as 2 sentences (with ".")

**Review by toaa-odoo — APPROVED** [review]

**Review by xpl-odoo — COMMENTED** [review]
Is there a reason for mixing plural and singular when discussing customer notes? I would use one or the other for the most part :)

- customer_notes.rst
- H1: Customer note
- H2 Add customer notes
- H3 Modify a customer note

**Comment by LoredanaLrpz** [comment]
> Is there a reason for mixing plural and singular when discussing customer notes? I would use one or the other for the most part :)
> 
> * customer_notes.rst
> * H1: Customer note
> * H2 Add customer notes
> * H3 Modify a customer note

Hey @xpl-odoo, "Customer Notes" is the name of the feature/setting; I use it in the plural form when I refer to it as the feature.
A customer note would be the concept of leaving a note; here, it's no longer a feature, so I left it as a noun.


However, this being said, I changed the name of the .rst file and added an "a" in the titles to have it all in singular (except for the one in the last § because it made more sense to leave it in the plural form)

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**